app/graphql/users/middlewares.js

```diff
@@ -10,5 +10,6 @@ module.exports = {
   // Here you add all the middlewares for the mutations, queries or field resolvers if you have any
   mutations: {
     createUser
-  }
+  },
+  fieldMiddlewares: { User: {} }
```

Comment:
this is necessary or you added it to show an example?

Reply:
It is an example of what to do if want to add field resolvers. Should I leave it there?

Reply:
leave it!